aiidalab_widgets_base/__init__.py

```diff
@@ -4,15 +4,17 @@
 from aiida import load_profile
 load_profile()
 
-from .viewers import viewer  # noqa
 from .codes import CodeDropdown, AiiDACodeSetup, valid_aiidacode_args  # noqa
 from .computers import SshComputerSetup, valid_sshcomputer_args  # noqa
 from .computers import AiidaComputerSetup, valid_aiidacomputer_args  # noqa
 from .computers import ComputerDropdown  # noqa
 from .databases import CodQueryWidget  # noqa
+#from .editors import editor  # noqa
 from .export import ExportButtonWidget  # noqa
 from .process import ProcessFollowerWidget, ProgressBarWidget, RunningCalcJobOutputWidget, SubmitButtonWidget  # noqa
 from .structures import StructureManagerWidget  # noqa
 from .structures import StructureBrowserWidget, StructureExamplesWidget, StructureUploadWidget, SmilesWidget  # noqa
 from .structures_multi import MultiStructureUploadWidget  # noqa
+from .viewers import viewer  # noqa
+
```

Comment:
Why is this pulled in to the root namespace as part of this PR?

Reply:
it is not pulled, I just ordered things alphabetically

Reply:
I mean, I didn't add it just now - I just replaced it.